modules/mobile/spec/models/adapters/appointments_vaos_v2_adapter_spec.rb

```diff
@@ -133,6 +134,15 @@ def appointment_by_id(id, overrides: {}, without: [])
   end
 
   describe 'appointment_type' do
+    context 'with appointment_type_consolidation flag on' do
+      before { Flipper.enable(:va_online_scheduling_appointment_type_consolidation) }
```

Comment:
allow(Flipper).to receive(:enabled?).with(: va_online_scheduling_appointment_type_consolidation).and_return(true)